torch/fx/examples/replace_op.py

```diff
@@ -0,0 +1,64 @@
+import torch
+from torch.fx import symbolic_trace
+import operator
+
+
+"""
+How to replace one op with another
+1. Iterate through all nodes in your GraphModule's Graph.
+2. Determine if the current node should be replaced. (Suggested: match
+on the node's ``target`` attribute).
+3. Create a replacement node.
+4. Use the fx built-in ``replace_all_uses_with`` to replace the current
+node with the replacement.
```

Comment:
Probably want to mention that this replaces the nodes "uses" rather than replacing the node itself. The current wording is ambiguous